### PR TITLE
Remove hover states

### PR DESCRIPTION
**Pull request by brodeur**

### CODE REVIEW

File: src/components/Account/AccountPopupContainer.tsx [hunk 1]
```diff
@@ -68,9 +72,16 @@ class AccountPopupContainer extends React.Component<IProps, null> {
   }
 
   public render() {
-    const { accountAddress, dao, profile, reputation } = this.props;
+    const { accountAddress, avatarSize, dao, detailTransferDetailView, profile, reputation } = this.props;
+
+    let className = " ";
+
+    if (detailTransferDetailView) {
```
Comment (tibetsprague) [inline src/components/Account/AccountPopupContainer.tsx]: you should use the classNames function like we do below for this

**Comment by jellegerbrandy** [comment]
@brodeur this PR is 20 days old, what is the status of it?

**Comment by tibetsprague** [comment]
@jellegerbrandy we decided to hold off on this for now, but might still want it later

**Comment by jellegerbrandy** [comment]
Ok @tibetsprague I'm closing this so we do not have to see it each time, we can always reopen when we need it....

**Pull request closed without merging**